web/packages/teleport/src/components/BannerList/StandardBanner.tsx

```diff
@@ -0,0 +1,73 @@
+/**
+ * Teleport
+ * Copyright (C) 2023  Gravitational, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React from 'react';
+
+import { Banner, Link } from 'design';
+
+import { CaptureEvent } from 'teleport/services/userEvent/types';
+import { userEventService } from 'teleport/services/userEvent';
+
+export type Severity = 'info' | 'warning' | 'danger';
+
+export type Props = {
+  message: string;
+  severity: Severity;
+  id: string;
+  link?: string;
+  onDismiss: (id: string) => void;
```

Comment:
IMO passing `id` to this callback is redundant, the parent component knows the id:
```ts
        <StandardBanner
          message={banner.message}
          severity={banner.severity}
          id={banner.id}
          link={banner.link}
          onDismiss={() => removeBanner(banner.id)}
          key={banner.id}
        />
```